### PR TITLE
[Merged by Bors] - feat: port Analysis.NormedSpace.Basic

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mattrobball** [comment]
Continuity is a still a little fiddly. Further diagnostics are a good idea before merging

File: Mathlib/Analysis/NormedSpace/Basic.lean [hunk 1]
```diff
@@ -163,132 +164,142 @@ theorem Filter.IsBoundedUnder.smul_tendsto_zero {f : ι → α} {g : ι → E} {
...
+    -- exact (continuous_const.add (continuous_norm.pow 2)).sqrt
   continuous_invFun := by
-    suffices ∀ y : ball (0 : E) 1, (1 - ‖(y : E)‖ ^ 2).sqrt ≠ 0 by continuity
+    suffices ∀ y : ball (0 : E) 1, (1 - ‖(y : E)‖ ^ 2).sqrt ≠ 0 by
+      /- Porting note: used to be continuity; should be restored. Has trouble with
+      Continuous.comp' being too eager -/
+      apply Continuous.smul (Continuous.inv₀
+        (continuous_const.sub ?_).sqrt this) continuous_induced_dom
+      continuity
+      -- apply (continuous_norm.comp continuous_induced_dom).pow 2
```
Comment (fpvandoorn) [inline Mathlib/Analysis/NormedSpace/Basic.lean:267]: remove comment?

**Review by fpvandoorn — APPROVED** [review]
LGTM, two minor comments

bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Comment by kmill** [comment]
The maintainers became aware of the personal (and hence non-professional) nature of the commit message inserted after the PR was delegated, and I am letting you know that we decided to force push to the master branch to remove it. This unfortunately disrupts a number of open PRs and complicates the porting process.

**Pull request closed without merging**